scripts/get_schemas.sh

```diff
@@ -51,7 +57,7 @@ for s in "${schemas[@]}"; do
   URL="${URL//_VERSION_/${VERSION}}"
 
 
-  FILEPATH="pkg/pulumiyaml/testing/test/testdata/${NAME}-${VERSION}.json"
+  FILEPATH="${dir}pkg/pulumiyaml/testing/test/testdata/${NAME}-${VERSION}.json"
```

Comment:
These files are not generated in the source tree anymore, but in the bazel directory structure, and thus we need to allow bazel to tell us where that is.  The script will still work as it is.